sci-misc/boinc/boinc-7.16.16.ebuild

```diff
@@ -86,16 +95,28 @@ pkg_setup() {
 }
 
 src_prepare() {
+	# bug #732024
+	if test "x$(get_libdir)" = "xlib64"; then
+	    sed -i -e 's,/:/lib:/usr/lib:,:/lib64:/usr/lib64:,g' m4/sah_check_lib.m4 || die
+	fi
+
 	default
 
 	# prevent bad changes in compile flags, bug 286701
 	sed -i -e "s:BOINC_SET_COMPILE_FLAGS::" configure.ac || die "sed failed"
 
 	eautoreconf
+
+	use X && setup-wxwidgets
```

Comment:
please put this back in `src_configure`

Reply:
Where does your diff come from? I do not have this when I do `git show`. And I had just resetted to gentoo/master --hard.

Reply:
Ah, I see. This ebuild is just a rename from boinc-7.16.14.ebuild. 
I'll do the changes you requested. Anything making those ebuilds is just fine. ;-)

Reply:
Done, works fine, thanks!

Reply:
Done, works fine, thanks!